tests/uts/test_client.py

```diff
@@ -82,20 +82,25 @@ def get_command_from_redis(self) -> dict:
         result = json.loads(job_data_raw.decode('utf-8'))
         return result
 
-    def test_run_command_is_written_to_redis(self):
-        cmd = self.client.run(self.host_id, 'bash', '-ce', 'echo hello')
-        sleep(0.1)  # write to redis must be completed before we pop from redis
+    @staticmethod
+    def generate_redis_answer(
+            id, cmd, line_timeout=None, log_file=None, max_output_per_channel=10485760,
+            set_new_logpath=None, timeout=3600.0):
+
+        return dict(
+            id=id, cmd=cmd, line_timeout=line_timeout, log_file=log_file,
+            max_output_per_channel=max_output_per_channel, set_new_logpath=set_new_logpath, timeout=timeout)
+
+    def assert_command_is_written_to_redis_properly(self, bash_command, **kwargs):
+        cmd = self.client.run(self.host_id, *bash_command, **kwargs)
         result = self.get_command_from_redis()
-        expected_value = {
-            'id': cmd.job_id,
-            'cmd': ['bash', '-ce', 'echo hello'],
-            'line_timeout': None,
-            'log_file': None,
-            'max_output_per_channel': 10485760,
-            'set_new_logpath': None,
-            'timeout': 3600.0}
+        expected_value = self.generate_redis_answer(id=cmd.job_id, cmd=bash_command, **kwargs)
         self.assertEqual(result, expected_value)
 
+    def test_run_command_is_written_to_redis(self):
```

Comment:
some of these tests / funcs names are too specific. I think this is more a `test_command_serialization`